analyzers/tests/ITs.JsonParser.Test/Sarif.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace ITs.JsonParser.Test;
+
+internal static class Sarif
+{
+    internal static string CreateReport(string rootPath, string solution, string project, string tfm, params string[] issues)
+    {
+        var content = $$"""
+            {
+              "runs": [
+                {
+                  "results": [
+                    {{string.Join($",{Environment.NewLine}", issues)}}
```

Comment:
We use `lf` on git level, so I would use it here as well, to make the git less confused when moving files around.

https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/expected/.gitattributes

Reply:
On my VS it says the file is LF, can you explain what you mean?

Reply:
Environment.NewLine in `\n\r` on my machine. 
So this would produce CrLf all over the place.
Update-expected.ps1 will move CrLf to `expected`. And this will confuse git to show all files as modified.
`expected` has .gitattributes with LF-only.
So staging those changes will change it to LF and will show only modified files.

So locally, some files will have LF (those pulled from server) and some files will have CR LF if they were created locally. That would be annoying to work with, so we should write it down as LF from the start.

Reply:
I am still a bit confused. 🤔
This is the file that generates the test dummy data for the UTs, which are not tracked by Git.
What do you mean about `update-expected.ps1`?
Is the comment on the wrong file or am I missing something?

Reply:
Ah, my bad. I've got confused, thanks for widening the context in your explanation. The test dummy is fine with Environment.NewLine